public/scripts/search.js

```diff
@@ -96,16 +90,16 @@ export default function initSearch(history, document, ordering, domUtils) {
         ordering.resetOrdering();
       }
     }
-
+    const result = searcher.search(query);
     let noResults = true;
     $icons.forEach(($icon) => {
-      const brandName = $icon.getAttribute('data-brand');
-      const score = getScore(query, brandName);
-      if (score < 0) {
+      const brandName = $icon.querySelector('h2').textContent;
```

Comment:
Can we use `$icon.getAttribute('data-brand')'` here like we did before?

Reply:
yes